src/main/webapp/app/entities/club-family/family-member-create.tsx

```diff
@@ -75,55 +116,80 @@ class FamilyMemberCreate extends React.Component<IFamilyMemberCreateProps> {
         </Row>
```

Comment:
Since this page included Add and Update
maybe can render the title(h2) based on create and update label

Reply:
Resolved in fcd046ac686a64140f789018c6a348f9f36a6866